mir_eval/key.py

```diff
@@ -24,26 +24,28 @@
 
 KEY_TO_SEMITONE = {'c': 0, 'c#': 1, 'db': 1, 'd': 2, 'd#': 3, 'eb': 3, 'e': 4,
                    'f': 5, 'f#': 6, 'gb': 6, 'g': 7, 'g#': 8, 'ab': 8, 'a': 9,
-                   'a#': 10, 'bb': 10, 'b': 11}
+                   'a#': 10, 'bb': 10, 'b': 11, 'x': -10}
```

Comment:
where does this `-10` value come from? Isn't this error-prone when computing keys mod 12 ?

Reply:
ok you are right!! @lostanlen

What do you think about this new implementation in 
6eb7bc2